src/main/java/com/github/wovnio/wovnjava/Headers.java

```diff
@@ -127,4 +128,20 @@ public boolean getIsValidRequest() {
         hreflangs.put(hreflangCodeDefaultLang, urlDefaultLang);
         return hreflangs;
     }
+
+    private boolean isIgnoredPath(String contextPath) {
+        ArrayList<String> ignorePaths = this.settings.ignorePaths;
+
+        contextPath = contextPath.toLowerCase();
+
+        for (String ignorePath : ignorePaths) {
+
+            String prefixPath = ignorePath;
+            String prefixPathTrailingSlash = ignorePath + "/";
+
+            if (contextPath.equals(prefixPath) || contextPath.startsWith(prefixPathTrailingSlash))
```

Comment:
It's a bit redundant to do `if(expression) return true`. Just do `return expression`